pypeman/channels.py

```diff
@@ -808,6 +808,33 @@ async def handle(self, msg):
         return result
 
 
+class MergeChannel(BaseChannel):
+    """
+    This class permits to have multiple watchers as inputs to a same channel
+
+    Args:
+        channels (list of channel watchers): List of channel watchers to use as inputs
```

Comment:
remove the word watchers if above questions confirm that this is generic

Reply:
Mmm, maybe we could call later, i thought the word watcher could apply also for mllp or http (I thought watcher more or less equal to listener)

Reply:
watcher is watcher,
listener is listener,
channel is channel,

and in the end this merges channels independently of the reason they're triggered.

socket listener,
file watcher,
timer channels,
...

Reply:
Valé
Done